### PR TITLE
Add day 2 muddy points

### PR DESCRIPTION
**Pull request by allyhawkins**
I added a notebook that addressed the few questions we got on day 2, in particular when to use and when not to use quotation marks.

### CODE REVIEW

File: supplemental_notebooks/2021-09-21_muddiest_points.Rmd [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+```{r}
+animals_df %>%
+  dplyr::filter(animals == cat)
+```
+You should see an error here and that's because it's searching for the variable cat, since we didn't put quotes around it to indicate that it was a string. 
+
+If we wanted to do it this way, we would have to make sure we assign `cat` to the string we are searching for first, and then filter. 
+
+```{r}
+cat = "cat"
```
Comment (jaclyn-taroni) [inline supplemental_notebooks/2021-09-21_muddiest_points.Rmd:96]: Nooo `cat()` is a built-in function that you will be causing conflict with. This is not a good practice even if the intended audience is unfamiliar with `cat`. Go with dog even though it hurts you.

File: supplemental_notebooks/2021-09-21_muddiest_points.Rmd [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+2. Navigate to RStudio server.  
+3. In the `File` pane, navigate to `training-modules/RNA-seq`.     
+4. Click the `Upload` button in the `File` pane.   
+5. Click `Choose file` and find this `Rmd` file on your computer, then click `OK`  
+
+
+## When to use quotations
+
+Some questions came up in the slack about when to use and when not to use quotations in R. 
+In general, the rule of thumb is that quotations are reserved for characters and strings (i.e. a word or set of words) only.
```
Comment (jaclyn-taroni) [inline supplemental_notebooks/2021-09-21_muddiest_points.Rmd:35]:
```suggestion
In general, the rule of thumb is that quotations are reserved for characters and strings (i.e., a word or set of words) only. 
```

**Review by jaclyn-taroni — APPROVED** [review]
Two comments. Don't use `cat` as a variable name and adding a comma. Once you make those updates and re-run, feel free to merge.

File: supplemental_notebooks/2021-09-21_muddiest_points.Rmd (hunk 1 above)
Comment (allyhawkins) [reply supplemental_notebooks/2021-09-21_muddiest_points.Rmd:96]: 😿 I forgot about that... I knew I was having too much fun.

**Pull request merged**